README.rst

```diff
@@ -31,20 +31,38 @@ Installation is simple::
 Usage
 -----
 
-Configure your database in ``settings.py`` from ``DATABASE_URL``::
+1. If database is already defined:
```

Comment:
I think maybe this should be "if ``DATABASES`` is already defined...", just to be clear what we mean. Does that make sense to you?

Also, what do you think about switching the order, so that the `DATABASES = {...}` becomes the first case presented? I think maybe that's more common, though I'm not totally sure.

Reply:
Thanks @jacobian  for reviewing the PR. I will make the changes requested.

I am also not sure about the order but can we go by this order only or I should make a change?

Reply:
There are other PRs also waiting for the review. Can you please review them too